src/test/java/Homework20.java

```diff
@@ -0,0 +1,35 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import java.time.Duration;
+public class Homework20 extends BaseTest {
+    @Test
+    public void deletePlaylist() throws InterruptedException {
+        /*openLoginUrl();*/
+        provideEmail("vera1077@gmail.com");
+        enterPassword("te$t$tudent");
+        clickSubmit();
+        selectPlaylist();
+        clickDeletePlaylist();
```

Comment:
Please, add one Assert

One test - One Assert

   **Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Reply:
Please review updated HW20